apps/wallet-dashboard/components/SupplyIncreaseVestingOverview.tsx

```diff
@@ -0,0 +1,149 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import { useCurrentAccount, useIotaClient } from '@iota/dapp-kit';
+import { useGetSupplyIncreaseVestingObjects } from '@/hooks';
+import {
+    ButtonType,
+    Card,
+    CardAction,
+    CardActionType,
+    CardBody,
+    CardType,
+    LabelText,
+    LabelTextSize,
+    Panel,
+    Title,
+} from '@iota/apps-ui-kit';
+import { StakeDialog, useStakeDialog } from './Dialogs';
+import { TIMELOCK_IOTA_TYPE, useCountdownByTimestamp, useFormatCoin } from '@iota/core';
+import { IOTA_TYPE_ARG } from '@iota/iota-sdk/utils';
+import SvgClock from '@iota/ui-icons/src/Clock';
+import { useQueryClient } from '@tanstack/react-query';
+
+export function SupplyIncreaseVestingOverview() {
+    const account = useCurrentAccount();
+    const address = account?.address || '';
+    const iotaClient = useIotaClient();
+    const queryClient = useQueryClient();
+    const { nextPayout, vestingSchedule, vestingMapped, vestingStakedMapped } =
+        useGetSupplyIncreaseVestingObjects(address);
+
+    const {
+        isDialogStakeOpen,
+        stakeDialogView,
+        setStakeDialogView,
+        selectedStake,
+        selectedValidator,
+        setSelectedValidator,
+        handleCloseStakeDialog,
+        handleNewStake,
+    } = useStakeDialog();
+
+    const formattedLastPayoutExpirationTime = useCountdownByTimestamp(
+        Number(nextPayout?.expirationTimestampMs),
+        false,
+        false,
+    );
+    const [formattedNextPayout, nextPayoutSymbol, nextPayoutResult] = useFormatCoin(
+        nextPayout?.amount,
+        IOTA_TYPE_ARG,
+    );
+
+    const [formattedAvailableStaking, availableStakingSymbol] = useFormatCoin(
+        vestingSchedule.availableStaking,
+        IOTA_TYPE_ARG,
+    );
+
+    const showSupplyIncreaseVestingOverview =
+        vestingMapped.length > 0 || vestingStakedMapped.length > 0;
+
+    function handleOnSuccess(digest: string): void {
+        iotaClient
+            .waitForTransaction({
+                digest,
+            })
+            .then(() => {
+                queryClient.invalidateQueries({
+                    queryKey: ['get-timelocked-staked-objects', account?.address],
+                });
+                queryClient.invalidateQueries({
+                    queryKey: [
+                        'get-all-owned-objects',
+                        account?.address,
+                        {
+                            StructType: TIMELOCK_IOTA_TYPE,
+                        },
+                    ],
+                });
+            });
+    }
+
+    return showSupplyIncreaseVestingOverview ? (
+        <div style={{ gridArea: 'vesting' }} className="with-vesting flex grow overflow-hidden">
+            <Panel>
+                <Title title="Vesting" />
+                <div className="flex h-full w-full items-center gap-md p-md--rs">
+                    <div className="w-1/2">
+                        <Card type={CardType.Filled} isHoverable>
```

Comment:
if the card isn’t meant to be clickable, it doesn’t need a hover effect :pray: 
```suggestion
                        <Card type={CardType.Filled} >
```